content/reference/cd-codefresh.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Codefresh
+aliases: ["cd-codefresh.html"]
```

Comment:
This line should be deleted. `cd-codefresh.html` didn't exist previously on the Jekyll-based site, so no need for an alias (redirect).

```suggestion
aliases: ["cd-codefresh.html"]
```